build/build_app_engine_package.py

```diff
@@ -80,8 +80,7 @@ def main(src_path, dest_path):
           shutil.copy(os.path.join(dirpath, name), dest_html_path)
     elif dirpath.endswith('app_engine'):
       for name in files:
-        if (name.endswith('.py') and name.find('test') == -1 or
```

Comment:
Everything in out/app_engine is pushed to the server during deployment. In general we don't want to push unnecessary files.